asreview/state/utils.py

```diff
@@ -200,6 +200,39 @@ def open_state(working_dir, review_id=None, read_only=True):
             pass
 
 
+def delete_state_from_project(project_path):
+
+    project_file_path = get_project_file_path(project_path)
+
+    try:
+        path_fm = get_feature_matrices_path(project_path)
+        shutil.rmtree(path_fm)
+    except Exception:
+        print("Failed to remove feature matrices.")
+
+    try:
+        path_review = get_reviews_path(project_path)
+        shutil.rmtree(path_review)
+    except Exception:
+        print("Failed to remove sql database.")
+
+    # open the projects file
+    with open(project_file_path, "r") as f_read:
+        project_dict = json.load(f_read)
+
+    # add path to dict (overwrite if already exists)
+    reset_config = {
+        'projectInitReady': False,
+        'reviewFinished': False,
+        'reviews': [],
+        'feature_matrices': []
+    }
```

Comment:
The key `projectHasDataset` has been deprecated? Now when the value of `reviews` is an empty array, it means no dataset?

Reply:
I don't think so. `projectHasDataset` is generated on the fly in 

https://github.com/asreview/asreview/blob/6af53348b67c16b0c6579a4871ff0b6137e351b2/asreview/webapp/api.py#L198-L203

`reviews` doesn't contain datasets. We decided to put only one dataset in a project, but with the option to have multiple reviews (useful for simulation in the future).